api/src/main/java/org/pmiops/workbench/actionaudit/Agent.java

```diff
@@ -0,0 +1,17 @@
+package org.pmiops.workbench.actionaudit;
+
+import org.pmiops.workbench.db.model.DbUser;
+
+public record Agent(AgentType agentType, Long idMaybe, String emailMaybe) {
```

Comment:
nice, thank you!

Reply:
so the old code and the parameter names here imply that the id and email are nullable...do we need to make them Optional in the java code? I'm not sure it would be necessary though...

Reply:
no, Long and String are already nullable.  You can see this in use in `asSystem()` - I can mark them as such to make it more obvious.